src/main/antora/modules/ROOT/pages/jdbc/query-methods.adoc

```diff
@@ -164,12 +164,12 @@ interface UserRepository extends CrudRepository<User, Long> {
 For converting the query result into entities the same `RowMapper` is used by default as for the queries Spring Data JDBC generates itself.
 The query you provide must match the format the `RowMapper` expects.
 Columns for all properties that are used in the constructor of an entity must be provided.
-Columns for properties that get set via setter, wither or field access are optional.
+Columns for properties that get set via setter or field access are optional.
```

Comment:
I am not sure about the `wither`, though I am almost sure it is a typo or defect, so I simply deleted it.

Reply:
It's not a typo. A "Wither" is similar to a getter but for immutable objects which returns an object wich is a copy of the original one but with a single property changed.


Something like 
```
record Person(String first, String last) {

    Person withLast(String last) {
        return new Person (this.first, last);
    }
}
```